userdocs/diagnostics/protocol-type-non-conformance.md

```diff
@@ -1,46 +1,59 @@
 
 ```swift
-protocol Word: Hashable {
-    var word: String { get }
-}
+print(Dog.species)    // Prints "Canus familiaris"
+print(Cat.species)    // Prints "Felis catus"
+print(Animal.species) // Error
+```
 
-struct Singular: Word {
-    var word: String
-}
+Since a type conforms to a protocol only when it satisfies _all_ of that protocol's requirements, the existential type `Animal` does not conform to the protocol `Animal` because it cannot satisfy its own requirement for the static property `species`:
 
-struct Plural: Word {
-    var word: String
+```swift
+func declareAnimalSpecies<T: Animal>(_ animal: T) {
+    animal.makeNoise()
+    print("My species is known as \(T.species)")
 }
 
-let singularWord = Singular(word: "mango")
-let pluralWord = Plural(word: "mangoes")
-
-let wordPairDict: [Word: Word] = [singularWord: pluralWord] // Error
+let dog = Dog()
+declareAnimalSpecies(dog)
+// Prints:
+// "Woof"
+// "My species is known as Canus familiaris"
+declareAnimalSpecies(animal)
+// Error: protocol type 'Animal' cannot conform to 'Animal'...
 ```
 
-One workaround to fix this problem is to use type erasure for the protocol `Word`. Think of type erasure as a way to hide an object's type. Since `Word` is of type `Hashable`, we already have `AnyHashable` type erasure available in the standard library which we can easily use here.
+In general, any initializers, static members, and associated types required by a protocol can be used only via conforming concrete types. Although Swift allows a protocol that requires initializers or static members to be used as a type, that type _does not and cannot_ conform to the protocol itself.
 
-```swift 
-// The fix
-let wordPairDict: [AnyHashable: AnyHashable] = [singularWord: pluralWord]
-```
+Currently, even if a protocol `P` requires no initializers or static members, the existential type `P` does not conform to `P` (with exceptions below). This restriction allows library authors to add such requirements (initializers or static members) to an existing protocol without breaking their users' source code.
+
+Concrete types that _do_ conform to protocols can provide functionality similar to that of existential types. For example, the standard library provides the `AnyHashable` type for `Hashable` values. Manual implementation of such __type erasure__ can require specific knowledge of the semantic requirements for each protocol involved and is beyond the scope of this discussion.
 
-# Exceptions
-`@objc` protocol type with no static requirements however do conform to its own protocol. Another exception is the `Error` Swift protocol.
+## Exceptions
 
+The Swift protocol `Error` has no requirements and, when used as a type, conforms to itself; `@objc` protocols with no static requirements can also be used as types that conform to themselves.
```

Comment:
While we're here, do you mind adding a link to https://docs.swift.org/swift-book/LanguageGuide/Protocols.html#ID275 at the end? That section must have been added in 5.3, I don't remember seeing it before...